front/components/assistant/RenderMessageMarkdown.tsx

```diff
@@ -140,18 +156,25 @@ export const CitationsContext = React.createContext<CitationsContextType>({
 });
 
 export function RenderMessageMarkdown({
+  owner,
   content,
   isStreaming,
   citationsContext,
   textSize,
   textColor,
 }: {
+  owner: WorkspaceType;
```

Comment:
I guess we don't need the full workspace type here.
```suggestion
  owner: LightWorkspaceType;
```